pkg/azurediskplugin/main.go

```diff
@@ -45,6 +45,7 @@ var (
 	driverName                 = flag.String("drivername", consts.DefaultDriverName, "name of the driver")
 	volumeAttachLimit          = flag.Int64("volume-attach-limit", -1, "maximum number of attachable volumes per node")
 	supportZone                = flag.Bool("support-zone", true, "boolean flag to get zone info in NodeGetInfo")
+	getNodeInfoFromLabels      = flag.Bool("get-node-info-from-labels", false, "boolean flag to get zone info from node labels in NodeGetInfo")
```

Comment:
I'm wondering if this flag should be set to true in the Windows image cli flags too

Reply:
I don't think so, we already have fall back in this PR which could ensure if IMDS does not work, get zone and instance type still works. I would work out another PR to support this setting in helm chart install.